components/base/sensorcontrolled/sensorcontrolled.go

```diff
@@ -3,10 +3,12 @@ package sensorcontrolled
 
```

Comment:
Can we move the MoveStraightFunctions into a `movestraight.go` file on this package?